config.go

```diff
@@ -59,5 +61,11 @@ func Parse(path string) (*Config, error) {
 		return nil, fmt.Errorf("packages in %s must be alphabetically ordered", path)
 	}
 
+	if c.GodocServer == "" {
+		c.GodocServer = "https://godoc.org"
```

Comment:
nit: This could be moved to a string const and reused in the tests:
```
const _defaultGodocServer = "https://godoc.org"
```

Reply:
Don't like the underscore to prefix the constants. But, it's just personal syntax sugar preference.